pages/mesosphere/dcos/services/kubernetes/2.4.10-1.15.10/supported-versions/index.md

```diff
@@ -30,6 +30,198 @@ DC/OS Kubernetes bundles the following versions of the base technology:
 Packages are versioned with an `a.b.c-x.y.z` format, where `a.b.c` is the version of the DC/OS integration and `x.y.z` indicates the version of Kubernetes.
 For example, `1.0.0-1.9.3` indicates version `1.0.0` of the DC/OS integration and version `1.9.3` of Kubernetes.
 
+# Debian packages installed by default
+
+The containers providing the Mesosphere Kubernetes Engine are based on Debian Buster. The control-plane nodes have these Debian packages included with the base image:
```

Comment:
@armandgrillet do the worker nodes have the same set of packages?

Reply:
Control plane, private, and public nodes use the `mesosphere/dcos-kubernetes-node` image.